Cabal/Distribution/PackageDescription/Check.hs

```diff
@@ -404,13 +423,15 @@ checkBenchmark _pkg bm =
 
     -- check that all autogen-modules appear on other-modules
   , check
-      (not $ and $ map
-        (flip elem (benchmarkModules bm))
-        (benchmarkModulesAutogen bm)
-      ) $
+      (not $ and $ map (flip elem (benchmarkModules bm)) (benchmarkModulesAutogen bm)) $
       PackageBuildImpossible $
              "On benchmark '" ++ prettyShow (benchmarkName bm) ++ "' an 'autogen-module' is "
           ++ "not on 'other-modules'"
+
+    -- check that all autogen-includes appear on includes
+  , check
```

Comment:
just wondering; is the code duplication (see diff chunk above) intentional?

Reply:
yes; click "expand above" and you see the duplication for checking `autogen-modules` for all component types.

Wont refactor in this PR.